packages/lab/stories/scrim/scrim.stories.tsx

```diff
@@ -1,134 +1,155 @@
-import { useRef, useState } from "react";
-import { Button } from "@salt-ds/core";
-import { ContentStatus, Scrim } from "@salt-ds/lab";
+import React, { useState } from "react";
+import { Button, Spinner, Text, H1, H2 } from "@salt-ds/core";
+import { Scrim } from "@salt-ds/lab";
 import { StoryFn, Meta } from "@storybook/react";
 
 export default {
   title: "Lab/Scrim",
   component: Scrim,
 } as Meta<typeof Scrim>;
 
+const Content = () => (
+  <>
+    <H1>What we do</H1>
+    <article id="main">
+      <section>
+        <H2>Investment Banking</H2>
+        <Text>
+          A global leader, we deliver strategic advice and solutions, including
+          capital raising, risk management, and trade finance to corporations,
+          institutions and governments.
+        </Text>
+      </section>
+      <section>
+        <H2>Markets</H2>
+        <Text>
+          Serving the world&apos;s largest corporate clients and institutional
+          investors, we support the investment cycle with market-leading
+          research, analytics and trade execution across multiple asset classes.
+        </Text>
+      </section>
+    </article>
+  </>
+);
+
+const exampleStyle: React.CSSProperties = {
+  position: "relative",
+  border: "1px solid grey",
+  maxWidth: 500,
+  padding: 24,
+};
+
+const buttonStyle: React.CSSProperties = {
+  marginTop: 30,
```

Comment:
instead of relying on a hard coded value here, can we use our layout components to achieve the same affect?

Reply:
will remove hardcoded values when updating examples